datasync/downloader/doc.go

```diff
@@ -18,5 +18,25 @@
 // This file is derived from eth/downloader/downloader.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package downloader contains the manual full chain synchronisation.
+/*
+Package downloader contains the manual full chain synchronisation.
+
+How downloader works
+
+The downloader is responsible for synchronizing up-to-date status from the peers connected to it. To do this,
+download "headers", "bodies", and "receipts" in parallel, merge them through the pipeline, and reflect them in the state trie.
+
+Source Files
+
+Downloader related functions and variables are defined in the files listed below
+  - api.go              : Console api to get synchronization information.
+  - downloader.go       : Functions and variables to sync peer and block. And modules for QOS.
+  - downloader_test.go  : Functions for testing the downloader package.
+  - events.go           : Define event type.
+  - metrics.go          : Metric Variables for Packet Transmission and Reception.
```

Comment:
`Metric Variables for Packet Transmission and Reception.`
->
`Metric variables for packet transmissions and receptions.`?